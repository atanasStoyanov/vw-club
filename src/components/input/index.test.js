import React from 'react';
import renderer from 'react-test-renderer';
import Input from '.';

describe('Input Component', () => {
    it('should render input field with label=Test', () => {
        const tree = renderer.create(<Input 
            label='Test'
            id='Test'
            value='Test'
            onChange='Test'
            type='text'
            placeholder='Test'
        />).toJSON();
        expect(tree).toMatchSnapshot();
    })
})