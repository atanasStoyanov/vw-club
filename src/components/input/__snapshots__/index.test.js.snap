// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input Component should render input field with label=Test 1`] = `
<div
  className="form-div"
>
  <label
    className="label"
    htmlFor="Test"
  >
    Test
    :
  </label>
  <input
    className="input"
    id="Test"
    onChange="Test"
    placeholder="Test"
    type="text"
    value="Test"
  />
</div>
`;
